torch/quantization/_correct_bias.py

```diff
@@ -0,0 +1,121 @@
+from __future__ import print_function, division, absolute_import
+import torch
+import torch.nn as nn
+import torch.nn.quantized as nnq
+
+import torch.quantization
+import torch.quantization._numeric_suite as ns
+
+_supported_modules = {nn.Linear, nn.Conv2d}
+_supported_modules_quantized = {nnq.Linear, nnq.Conv2d}
+
+def get_module(model, name):
+    ''' Given name of submodule, this function grabs the submodule from given model
+    '''
+    current = model
+    name = name.split('.')
+    for subname in name:
+        if subname == '':
+            return current
+        current = current._modules[subname]
+    return current
+
+def get_parent_module(model, name):
```

Comment:
this can be come get_parent_name.
I have some code from GraphModule that will be useful here
```python
# turn foo.bar -> ['foo', 'bar']
def _parent_name(target):
    r = target.rsplit('.', 1)
    if len(r) == 1:
        return '', r[0]
    else:
        return r[0], r[1]
```